scripts/mods/appliedenergistics_singularity.zs

```diff
@@ -0,0 +1,100 @@
+#modloaded zenutils appliedenergistics2
```

Comment:
No, return `#sideonly` back! Its was right to add tooltips and "hint" recipes only client side. Ideally, all my scripts that working with tooltips and JEI should be clientside only.
To fix recipe, just put it back in `scripts/mods/appliedenergistics.zs`.

Reply:
done, the "hint" crafting recipe is also added st server side, because I'm not sure about if adding only for client will cause data syncing issue. (It happened before)

Reply:
You just removed it again by merging